examples/dt/uni01alpha/networker/values.yaml

```diff
@@ -0,0 +1,139 @@
+        edpm_network_config_os_net_config_mappings:
+          edpm-networker:
+            nic2: enp2s0
+
+        edpm_enable_chassis_gw: true
+        edpm_ovn_availability_zones:
+          - zone-2
+
+        edpm_sshd_configure_firewall: true
+        edpm_sshd_allowed_ranges:
+          - '192.168.122.0/24'
+
+        edpm_network_config_template: |
+          ---
+          {% set mtu_list = [ctlplane_mtu] %}
+          {% for network in role_networks %}
+          {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+          {%- endfor %}
+          {% set min_viable_mtu = mtu_list | max %}
+          network_config:
+            - type: ovs_bridge
+              name: {{ neutron_physical_bridge_name }}
+              mtu: {{ min_viable_mtu }}
+              use_dhcp: false
+              dns_servers: {{ ctlplane_dns_nameservers }}
+              domain: {{ dns_search_domains }}
+              addresses:
+                - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+              routes: {{ ctlplane_host_routes }}
+              members:
+                - type: interface
+                  name: nic1
+                  mtu: {{ min_viable_mtu }}
+                  # force the MAC address of the bridge to this interface
+                  primary: true
+          {% for network in nodeset_networks %}
+              - type: vlan
+                mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+                vlan_id: {{ lookup('vars', networks_lower[network] ~ '_vlan_id') }}
+                addresses:
+                  - ip_netmask: {{ lookup('vars', networks_lower[network] ~ '_ip') }}/{{ lookup('vars', networks_lower[network] ~ '_cidr') }}
+                routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+          {% endfor %}
+
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth0
+
+    networks:
+      - defaultRoute: true
+        name: ctlplane
+        subnetName: subnet1
+      - name: internalapi
+        subnetName: subnet1
+      - name: tenant
+        subnetName: subnet1
+
+    nodes:
+      edpm-networker-0:
+        hostName: edpm-networker-0
+        ansible:
+          ansibleHost: 192.168.122.105
+        networks:
+          - name: ctlplane
+            subnetName: subnet1
+            defaultRoute: true
+            fixedIP: 192.168.122.105
+          - name: internalapi
+            subnetName: subnet1
+          - name: tenant
+            subnetName: subnet1
+      edpm-networker-1:
+        hostName: edpm-networker-1
+        ansible:
+          ansibleHost: 192.168.122.106
+        networks:
+          - name: ctlplane
+            subnetName: subnet1
+            defaultRoute: true
+            fixedIP: 192.168.122.106
+          - name: internalapi
+            subnetName: subnet1
+          - name: tenant
+            subnetName: subnet1
+      edpm-networker-2:
+        hostName: edpm-networker-1
```

Comment:
edpm-networker-2

Reply:
Thank you. Done. The latest commit would have  the fix.